jobs/pn-consumers/package.json

```diff
@@ -25,13 +25,14 @@
   },
   "dependencies": {
     "@aws-sdk/client-s3": "^3.387.0",
+    "@interop-be-reports/commons": "workspace:*",
     "axios": "^1.4.0",
     "chalk": "^5.3.0",
     "dotenv": "^16.3.1",
     "lodash": "^4.17.21",
     "mongodb": "^5.7.0",
+    "mongodb-memory-server": "^8.15.1",
```

Comment:
shouldn't this be under `devDependencies`?

Reply:
👆